configuration/africas-talking.md

```diff
@@ -6,23 +6,26 @@ SMS messages can be sent and received using the [Africa's Talking](https://afric
 
 ## Africa's Talking configuration
 
+First generate a long unique key to use as the `medic-api-key`.
+
 Log on to the [Africa's Talking Dashboard](https://account.africastalking.com) and configure your callback URLs as follows.
 
-- Delivery Reports: `https://<hostname>/api/v1/sms/africastalking/delivery-reports`
-- Incoming Messages: `https://<hostname>/api/v1/sms/africastalking/incoming-messages`
+- Delivery Reports: `https://<hostname>/api/v1/sms/africastalking/delivery-reports?key=<medic-api-key>`
+- Incoming Messages: `https://<hostname>/api/v1/sms/africastalking/incoming-messages?key=<medic-api-key>`
 
-Then generate an "API Key" and save this in your Medic configuration covered below.
+Then generate an "API Key" (we'll refer to this as the `at-api-key`) and save this in your Medic configuration covered below.
 
 ## Medic configuration
 
-### API key
+### API keys
 
-The API key should be treated as securely as a password as anyone with access to it will be able to send messages using your account. Therefore we store it in the CouchDB configuration.
+The API keys should be treated as securely as a password as anyone with access to them will be able to send messages using your account. Therefore we store them in the CouchDB configuration.
 
-To add the credential to the admin config you need to either [PUT the value using curl](https://docs.couchdb.org/en/stable/api/server/configuration.html#put--_node-node-name-_config-section-key) or similar:
+To add the credentials to the admin config you need to either [PUT the value using curl](https://docs.couchdb.org/en/stable/api/server/configuration.html#put--_node-node-name-_config-section-key) or similar:
 
 ```sh
-curl -x PUT http://admin:pass@localhost:5984/_node/couchdb@127.0.0.1/_config/medic-credentials/africastalking.com -d "<your-api-key>"
+curl -X PUT https://<user>:<pass>@<domain>/_node/couchdb@127.0.0.1/_config/medic-credentials/africastalking.com:incoming -d '"<medic-api-key>"'
+curl -X PUT https://<user>:<pass>@<domain>/_node/couchdb@127.0.0.1/_config/medic-credentials/africastalking.com:outgoing -d '"<at-api-key>"'
```

Comment:
This update is missing in the "You can also add it via Fauxton" section below.

Reply:
Thanks! Updated.